test/integration/connect/envoy/docker-compose.yml

```diff
@@ -46,24 +46,48 @@ services:
     depends_on:
       - consul
     image: "fortio/fortio"
+    environment:
+      - "FORTIO_NAME=s1"
     command:
      - "server"
      - "-http-port"
      - ":8080"
      - "-grpc-port"
      - ":8079"
+     - "-redirect-port"
```

Comment:
This wasn't used and it was only generating an error when it collided with other fortio instances.